src/config.py

```diff
@@ -0,0 +1,8 @@
+"""forum123's configuration module."""
+import os
```

Comment:
Some files have one empty line between docstring and imports, some have two, some have zero. To make it consistent let's put one empty line between docstring and imports everywhere.

It means that we need to fix this file and `forms.py` as well.